app/Livewire/Server/Show.php

```
@@ -361,6 +374,87 @@ public function instantSave()
         }
     }
 
+    public function checkHetznerServerStatus(bool $manual = false)
```

PHPMD (2.15.0)

The method checkHetznerServerStatus has a boolean flag argument $manual, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)